libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -132,13 +132,23 @@ bool ofShader::load(string shaderName) {
 
 //--------------------------------------------------------------
 bool ofShader::load(string vertName, string fragName, string geomName) {
-	if(vertName.empty() == false) setupShaderFromFile(GL_VERTEX_SHADER, vertName);
-	if(fragName.empty() == false) setupShaderFromFile(GL_FRAGMENT_SHADER, fragName);
+	if(vertName.empty() == false) {
```

Comment:
won't this be better like?

```cpp
if(!vertName.empty() && !setupShaderFromFile(GL_VERTEX_SHADER, vertName){
    return false;
}
```

so many comparisons with false look confusing but otherwise seems good